packages/generator/gulpfiles/downloadRamlsTask.ts

```diff
@@ -93,13 +91,30 @@ function downloadRamlFromExchange(): Promise<void> {
   });
 }
 
-gulp.task("cleanInputDir", async () => {
-  return fs.removeSync(path.join(config.inputDir));
+gulp.task("diffRamlFiles", async () => {
+  const result = await diffNewAndArchivedRamlFiles(
+    config.apiBackupDir,
+    config.inputDir,
+    path.join(config.apiBackupDir, config.apiConfigFile),
+    path.join(config.inputDir, config.apiConfigFile)
+  );
+  return fs.writeFile(config.diffFile, JSON.stringify(result));
+});
+
+gulp.task("cleanup", async () => {
+  fs.removeSync(path.join(config.apiBackupDir));
+  fs.removeSync(config.diffFile);
+  return fs.moveSync(
```

Comment:
Why is cleanup moving this directory? Naming stuff is hard but I think this task would probably be better off named "backupApis" or "preUpdateApis". cleanup can get confused with a clean task which this is not.